randovania/games/prime2/logic_database/Agon Wastes.json

```diff
@@ -2156,7 +2156,7 @@
                     "node_type": "dock",
                     "heal": false,
                     "coordinates": null,
-                    "description": "https://www.youtube.com/watch?v=-LZluWe8QvM&list=PLevfta9o_QKWuQNXAnmBXtzW4nmU5VLyN&index=46",
+                    "description": "<https://youtu.be/Myr8MRM5ZDc&t=88>",
```

Comment:
whats the deal with having the links in `<>`?

Reply:
when I updated the links, some of them lost the ability to hyperlink for some reason.
the referred to other instances and they had angle bracket and that seems to fix them

Reply:
oh huh. weird but ok